### DISCUSSION TITLE
DIY Tank Method 2 Questions

### DISCUSSION THREAD

**Discussion by bcrazycramer** [jgyates/genmon#1074] [General]
Hello,

I would like to move away from using the Mopeka sensors to monitor LP fuel levels. 

I'd rather buy a premade cable that will allow me to slip it on the top of the Remote Ready Dial. Can I purchase this Rochester 9701 Right Angle Module and then continue the DIY Tank Method 2 with the ADC board and Raspberry Pi 4?

Thanks,
Brandon

**Comment by jgyates** [comment]
If you have not already, look over this page:  https://github.com/jgyates/genmon/wiki/Appendix-L-Adding-a-Propane-Fuel-Gauge-to-Genmon

@curtis1757 may be able to better answer your question on the DIY Method 2 approach.

**Reply by bcrazycramer** [reply]
Looking at this now, not sure why I was down in the DIY2 section? Need more sleep I guess...

FYI

Rochester must have changed their domain name. Most of the links are broken because they have rochestergauges.com instead of rochestersensors.com. 

Thanks,
Brandon

**Reply by jgyates** [reply]
Thanks for letting me know. I updated the links to point to rochestersensors.com
